index.d.ts

```diff
@@ -1,6 +1,15 @@
 export namespace Temporal {
   export type ComparisonResult = -1 | 0 | 1;
-  export type RoundingMode = 'halfExpand' | 'ceil' | 'trunc' | 'floor';
+  export type RoundingMode =
```

Comment:
Looks like we also need to make this change in proposal-temporal. @12wrigja do you want to PR this?  If not then I can do it.

Reply:
I can send it - thanks for the reminder.

Reply:
Opened https://github.com/tc39/proposal-temporal/pull/2489